binding/rust/README.md

```diff
@@ -57,12 +57,7 @@ use rhino::RhinoBuilder;
 let rhino: Rhino = RhinoBuilder::new("/path/to/context/file.rhn").init().expect("Unable to create Rhino");
 ```
 The context file is a Speech-to-Intent context created either using
-[Picovoice Console](https://picovoice.ai/console/) or one of the default contexts available on Rhino's GitHub repository:
-```rust
-use rhino::{BuiltinContexts, RhinoBuilder};
-
-let rhino: Rhino = RhinoBuilder::new_with_builtin(BuiltinContext::CoffeeMaker).init().expect("Unable to create Rhino");
-```
+[Picovoice Console](https://picovoice.ai/console/) or one of the default contexts available on Rhino's GitHub repository.
```

Comment:
maybe have "Rhino's GitHub repository" link to the `contexts` folder in the repo